azure-mgmt-redis/src/main/java/com/microsoft/azure/management/redis/RedisAccessKeys.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.redis;
+
+import com.microsoft.azure.management.redis.implementation.RedisAccessKeysInner;
+
+/**
+ * The {@link RedisCache#keys} action result.
+ */
+public class RedisAccessKeys {
+    private RedisAccessKeysInner inner;
+
+    /**
+     * Creates an instance of the Redis Access keys result object.
+     *
+     * @param inner the inner object
+     */
+    public RedisAccessKeys(RedisAccessKeysInner inner) {
```

Comment:
why is this constructor public? Is this class intended to be instantiated by the user?  If it's only purpose is to be instantiated by our own wrappers inside the same package namespace, then it should be package-internal (no `public` keyword)